registration_test.go

```diff
@@ -247,3 +249,61 @@ func TestUnregister(t *testing.T) {
 		})
 	}
 }
+
+func TestInstall(t *testing.T) {
+	if wsl.MockAvailable() {
+		t.Skip("Skipping because the mock does not capture the compexity of this function")
```

Comment:
hum, we can still run with the dummy mock, no? The one where you can return an error or nothing, so I wouldn’t skip it if possible. In general, I would at least exercise all the code, even if the mock doesn’t really reflect the reality, what do you think?